strings/base_activation.h

```diff
@@ -30,15 +34,8 @@ namespace winrt::impl
 
         if (hr == impl::error_not_initialized)
         {
-            auto usage = reinterpret_cast<int32_t(__stdcall*)(void** cookie) noexcept>(WINRT_IMPL_GetProcAddress(load_library(L"combase.dll"), "CoIncrementMTAUsage"));
-
-            if (!usage)
-            {
-                return hr;
-            }
-
             void* cookie;
-            usage(&cookie);
+            WINRT_IMPL_CoIncrementMTAUsage(&cookie);
```

Comment:
[preexisting] This is not checking the return value of `CoIncrementMTAUsage`.  Nor does it ever call `CoDecrementMTAUsage`.  I'm not sure if that is a problem or not.